doc/faq/a05-board-generator.rst

```diff
@@ -0,0 +1,80 @@
+:orphan:
+
+Board generator
+---------------
+
+The board generator is a python script originally intended to ease the
+Arduino IDE's `boards.txt` configuration file about the multitude of
+available board, especially when common data have to be updated for all of
+them.
+
+This script is also used to manage uncommon options that will not find their
+place into the IDE menu.
+
+-  `How can I run the script ? <#how-can-i-run-the-script>`__
+-  `What can I do with it ? <#what-can-i-do-with-it>`__
+-  `When do I need to update it ? <#when-do-i-need-to-mess-with-it>`__
+-  `Why is my pull-request failing continuous-integration ? <#why-is-my-pull-request-failing-continuous-integration>`__
+
+How can I run the script ?
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Python 2 needs to be installed on your system.
```

Comment:
Does it work with Python 3?

Reply:
> Does it work with Python 3?

No. At least all `print`s must be updated.